Procfile

```diff
@@ -1 +1 @@
-web: npm start
+web: npm run start:prod
```

Comment:
This change would restrict other deployments, that use a `Procfile`, to running in "production" mode. The NODE_ENV configuration should be responsible for determining the mode the application is running in. There can be times that other environment configurations would want to be used; development, test, stage, qa, etc.

I understand that by clicking the "Deploy to Heroku" button for the first time, would run the app with the default development configuration. However, if an app by the name specified in `app.json` already exists, then the NODE_ENV can be set.

Reply:
In fact, if you change the npm script to use the NODE_ENV, even the first time you deploy it will be ok, because I set the NODE_ENV to production.
I will revert the proc file but let you change the script.

Reply:
Is this the script you're referring to? https://github.com/meanjs/mean/blob/master/gulpfile.js#L484

I plan to remove the `env:dev` task from the sequence.

Reply:
You said: "If I remove the 'env:dev' from the default Gulp task, the application runs in whatever NODE_ENV I set in my environment variables." 
So I think that was it.